bindings/nodejs/lib/types/client/bridge/index.ts

```diff
@@ -49,7 +49,7 @@ import type {
     __BuildNftOutputMethod__,
     __ClearListenersMethod__,
     __SignatureUnlockMethod__,
-    __MinimumRequiredStorageDepositMethod__,
+    __MinimumRequiredOutputAmountMethod__,
```

Comment:
This needs to be changed to `__ComputeMinimumOutputAmountMethod__`
Same for the second one.